android/build.gradle

```diff
@@ -34,6 +38,17 @@ repositories {
 }
 
 dependencies {
-    compile 'com.android.support:support-media-compat:26.+'
-    compile "com.facebook.react:react-native:+"
+    implementation 'com.android.support:support-media-compat:26.+'
+    provided "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    implementation("com.github.bumptech.glide:glide:${safeExtGet("glideVersion", "4.7.1")}") {
+        exclude group: "com.android.support", module: "glide"
+    }
+    implementation("com.github.bumptech.glide:annotations:${safeExtGet("glideVersion", "4.7.1")}") {
+        exclude group: "com.android.support", module: "annotations"
+    }
+    annotationProcessor "com.github.bumptech.glide:compiler:${safeExtGet("glideVersion", "4.7.1")}"
+    implementation("com.github.bumptech.glide:okhttp3-integration:${safeExtGet("glideVersion", "4.7.1")}") {
+        exclude group: "com.android.support", module: "okhttp3-integration"
+        exclude group: "com.android.support", module: "glide-parent"
+    }
```

Comment:
Why do you need this many Glide dependencies?

Reply:
follow glide's official integration steps

Reply:
We may not want to force glide onto consumers of this library, especially since React Native already ships with Fresco. 

Could we use the Fresco equivalent instead?